src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -236,15 +236,14 @@ void GcsActorManager::HandleGetNamedActorInfo(
   ActorID actor_id = GetActorIDByName(name, ray_namespace);
 
   Status status = Status::OK();
-  if (actor_id.IsNil()) {
-    // The named actor was not found.
+  const auto &iter = registered_actors_.find(actor_id);
```

Comment:
actually & here doesn't make sense. find will return an iterator, not a r-value. `auto iter =` should just work.

Reply:
It seems to be used everywhere btw... I modified 2 places including this one.